src/status_im/chat/events.cljs

```diff
@@ -290,12 +295,12 @@
 (defn start-chat
   "Start a chat, making sure it exists"
   [chat-id opts {:keys [db] :as cofx}]
-  ; don't allow to open chat with yourself
+                                        ; don't allow to open chat with yourself
```

Comment:
something is wrong with indentation here

Reply:
Fixed

Reply:
is it?